Kitodo/src/main/java/org/kitodo/production/forms/createprocess/FileUploadDialog.java

```diff
@@ -70,6 +73,22 @@ public void handleFileUpload(FileUploadEvent event) {
 
             LinkedList<TempProcess> processes = new LinkedList<>();
             processes.add(tempProcess);
+
+            Collection<String> higherLevelIdentifier = this.createProcessForm.getRulesetManagement()
```

Comment:
The method `handleFileUpload` gets a little bloated - could you move this new part to a separate private method that is called instead to improve code readability?